ProfileManager.js

```diff
@@ -115,6 +125,82 @@ export default class ProfileManager {
     return await this._getAgentContent({profileAgentRecord});
   }
 
+  /**
+   * Gets the profile agent assigned to the account associated with the
+   * authenticated session for the profile identified by the given profile ID.
+   *
+   * @param {object} options - The options to use.
+   * @param {string} options.profileId - The ID of the profile to get the
+   *   profile agent for.
+   * @param {string} options.capabilityInvoker - The type of invoker to sign
+   * capability invocations.
+   *
+   * @returns {Promise<object>} The profile agent.
```

Comment:
This function is called `getCapability` ... these docs seem to have been copy and pasted without being fully updated.

Reply:
No shame here, they were. I will update it to the proper description.